### PR TITLE
Move Ajna token address to deployer, RWT improvements

### PR DESCRIPTION
**Pull request by EdNoepel**
Per discussion this morning, moved Ajna token address to `PoolDeployer`, such that it's in one place instead of three.  Will still need a solution to parameterize this for deployment to other chains.

Unrelated, some minor improvements to the real-world test.

### CODE REVIEW

File: src/base/ScaledPool.sol [hunk 1]
```diff
@@ -72,7 +72,7 @@ abstract contract ScaledPool is Clone, FenwickTree, Multicall, IScaledPool {
     /**
      *  @notice Address of the Ajna token, needed for Claimable Reserve Auctions.
      */
-    address internal ajnaTokenAddress = 0x9a96ec9B57Fb64FbC60B423d1f4da7691Bd35079;
+    address internal ajnaTokenAddress = address(0);
```
Comment (MikeHathaway) [inline src/base/ScaledPool.sol]: Doesn't the address already default to the 0 address without needing to be assigned?

Comment (grandizzy) [inline src/base/ScaledPool.sol]: assignment can be removed as the default value for address type is 0x

**Review by grandizzy — APPROVED** [review]
LGTM, minor comment re assigning 0x address

**Review by MikeHathaway — APPROVED** [review]
LGTM

File: src/base/ScaledPool.sol (hunk 1 above)
Comment (EdNoepel) [reply src/base/ScaledPool.sol]: Addressed in 4d21b29bc86e4d16ff8c87408f28109b8985fbf1.

Comment (EdNoepel) [reply src/base/ScaledPool.sol]: Yes; addressed in 4d21b29bc86e4d16ff8c87408f28109b8985fbf1.

**Pull request merged**